src/index.ts

```diff
@@ -7,6 +7,7 @@ const version = require("../package.json").version;
 
 // Expose all types
 export * from "./soroban_rpc";
+export * from "./contract_spec";
```

Comment:
I had been using it to export some stuff for tests since it was much easier to construct the example ScSpecTypeDef types by hand. I can move it to the tests and switch to the singe output so it's a smaller API surface.

Reply:
Totally fair re: testing!